devices/sonoff/trvzb-thermostat.json

```diff
@@ -0,0 +1,103 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "SONOFF",
+  "modelid": "TRVZB",
+  "vendor": "SONOFF",
+  "product": "TRVZB Thermostat",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_THERMOSTAT",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0201"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0301",
+        "endpoint": "0x01",
+        "in": [
+          "0x0000",
+          "0x0201"
+        ]
+      },
```

Comment:
In order to make use of the `config/mode` item, the following code must be added and contain the actually usable settings (see respetive item comment)

```
      "meta": {
        "values": {
          "config/mode": {"off": 0, "auto": 1, "heat": 4}
        }
      },
```